packages/hooks/src/base.ts

```diff
@@ -44,52 +44,49 @@ export class HookManager {
   }
 
   getMiddleware (): Middleware[]|null {
-    const previous = this._parent?.getMiddleware();
+    const previous = this._parent && this._parent.getMiddleware();
```

Comment:
I wonder why this is different. It should be the same thing - only thing I can think of is that the compilation target was too new (Node > 14).